File: Python/jewels-and-stones.py
# https://leetcode.com/problems/jewels-and-stones/

class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        count = 0
        for j in J:
            for s in S:
                if j == s:
                    count = count + 1
        return count


o = Solution()
print(o.numJewelsInStones("aA", "aAAbbbb")) # 3
print(o.numJewelsInStones("z", "ZZ")) # 0

File: JavaScript/jewels-and-stones.js
/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */
var numJewelsInStones = function(J, S) {
    for (var i = 0, count = 0; i < S.length; i++) {
        if(J.indexOf(S.charAt(i)) !== -1) {
            count++;
        }
    }
    return count
};

console.log(numJewelsInStones("aA", "aAAbbbb"));